lib/app/include/ocall.h

```diff
@@ -6,4 +6,9 @@
 #define OCALL_MEMORY_FREE              2002
 #define OCALL_SYSCALL              2003
 #define OCALL_CACHE_BINDING    2004
-#define OCALL_CACHE_UNBINDING  2005
\ No newline at end of file
+#define OCALL_CACHE_UNBINDING  2005
+
+#define ENCLAVE_DEFAULT_KBUFFER_SIZE              0x1000UL
```

Comment:
I suggest to add comments here to briefly explain what's the kbuffer (i.e., for passing arguments/results in the case of ocall).